test/tests/kernels/ad_mat_coupled_annihilation/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [ad_mat_coupled_annihilation]
+    type = 'Exodiff'
+    input = 'ad_mat_coupled_annihilation_test.i'
+    exodiff = 'ad_mat_coupled_annihilation_test_out.e'
+    issues = '#35'
+    design = '/ADMatCoupledDefectAnnihilation.md'
+    requirement = 'The system shall compute the annihilation of variable c_b towards its equilibrium concentration in the material.'
```

Comment:
```suggestion
    requirement = 'The system shall compute the annihilation of a concentration variable towards its equilibrium in the material.'
```

This is a really good improvement, but I would remove specific variable names (unless you want to define it, which might end up making the requirement long).